api/v1alpha/src/admin/bq/procedure/permissionsDiff.sql

```diff
@@ -0,0 +1,66 @@
+CREATE OR REPLACE PROCEDURE `${projectId}.datashare.permissionsDiff`(policyFilter ARRAY<STRING>)
+BEGIN
+WITH ranked AS (
+  -- Rank all of the policies by date descending. The latest record will be 1.
+  SELECT
+    p.*,
+    DENSE_RANK() OVER (PARTITION BY policyId ORDER BY createdAt desc) AS rank
+  FROM `${projectId}.datashare.policy` p
```

Comment:
Can this be a dynamic variable from the config for `dataset.table`? Same for line 26